### PR TITLE
adding 'brand' data from Dohánybolt

### LINKED ISSUE #1811: Research needed for: shop/tobacco|Dohánybolt
#### :thinking: We need help deciding what to do with "shop/tobacco|Dohánybolt"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/tobacco|Dohánybolt` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/tobacco|Dohánybolt`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/tobacco|Dohánybolt`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by good-praxis**
fixes #1811 
[source](https://en.wikipedia.org/wiki/Doh%C3%A1nybolt)

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -32383,19 +32383,17 @@
...
-    "match": ["shop/tobacco|Nemzeti dohánybolt"],
+    "match": [
+      "shop/tobacco|Dohánybolt",
+      "shop/tobacco|Nemzeti dohánybolt"
+    ],
     "tags": {
       "brand": "Nemzeti Dohánybolt",
+      "brand:wikidata": "Q20639040",
+      "brand:wikipedia": "en:Dohánybolt",
+      "countryCodes": ["hu"],
```
Comment (ENT8R) [inline config/canonical.json]: This has to be below the `count` tag...

**Comment by ENT8R** [comment]
BTW: Google translates it directly to tobacco shop (https://translate.google.de/#auto/en/doh%C3%A1nybolt) Are you sure that this is no generic name after all?

**Comment by good-praxis** [comment]
It is _kind of_ generic. Expect for the fact that they're all state run, so the same actor is behind them, just like a commerical brand. However, this is probably worth a bigger debate

**Comment by bhousel** [comment]
This is old, so I'm just going to do what @ImreSamu suggested here: 
https://github.com/osmlab/name-suggestion-index/issues/2028#issuecomment-431662331

**Pull request closed without merging**